Description: Use sensible-browser
 This patch enables Dia to use Debian's sensible-browser
Author: Roland Stigge <stigge@antcom.de>

--- dia-0.97.1.orig/app/commands.c
+++ dia-0.97.1/app/commands.c
@@ -690,7 +691,7 @@ help_manual_callback (GtkAction *action)
   ShellExecuteA (0, "open", helpindex, NULL, helpdir, SW_SHOWNORMAL);
 #else
   command = getenv("BROWSER");
-  command = g_strdup_printf("%s 'file://%s' &", command ? command : "xdg-open", helpindex);
+  command = g_strdup_printf("%s 'file://%s' &", command ? command : "sensible-browser", helpindex);
   system(command);
   g_free(command);
 #endif

